app/services/idv/steps/in_person/state_id_step.rb

```diff
@@ -24,6 +24,10 @@ def call
           formatted_dob = MemorableDateComponent.extract_date_param flow_params&.[](:dob)
           pii_from_user[:dob] = formatted_dob if formatted_dob
 
+          add_form_analytics_properties_to_flow_session(
+            { birth_year: flow_params[:dob][:year] },
+          )
```

Comment:
We can return a `FormResponse` from this method and it will be passed to analytics, so I think we could do

```ruby
    # last part of this method
    FormResponse.new(success: !@redirect, errors: {}, extra: { birth_year: flow_params[:dob][:year] }
  end
```


https://github.com/18F/identity-idp/blob/e39eb3e2071a9aa6647e93a233e5230273a94ccd/app/services/flow/base_flow.rb#L69-L78

Reply:
Also another thought... what if we did `parsed_dob&.year`?

Reply:
I will look into using the `FormResponse`. That seems like a great way to handle this.

Reply:
I updated the `state_id_step#call` to return a `FromResponse` with the birth year in the extra hash param.